src/portal/src/app/project/create-project/create-project.component.html

```diff
@@ -58,6 +58,45 @@ <h3 class="modal-title">{{'PROJECT.NEW_PROJECT' | translate}}</h3>
                     </clr-control-error>
                 </div>
             </div>
+            <div class="clr-form-control" *ngIf="isSystemAdmin">
+                <label for="create_project_storage_limit" class="clr-control-label">{{ 'PROJECT.PROXY_CACHE' | translate }}
+                    <clr-tooltip>
+                        <clr-icon clrTooltipTrigger shape="info-circle" size="24"></clr-icon>
+                        <clr-tooltip-content clrPosition="bottom-left" clrSize="lg" *clrIfOpen>
+                            <span>{{ 'PROJECT.PROXY_CACHE_TOOLTIP' | translate }}</span>
+                        </clr-tooltip-content>
+                    </clr-tooltip>
+                </label>
+                <clr-toggle-wrapper class="mt-02">
+                        <input type="checkbox" clrToggle name="proxy-cache" id="proxy-cache"
+                               [(ngModel)]="enableProxyCache"/>
+                </clr-toggle-wrapper>
+                <div *ngIf="enableProxyCache" class="clr-select-wrapper ml-1">
+                    <select class="width-164" id="registry" name="registry" [(ngModel)]="project.registry_id">
+                        <option class="display-none" value=""></option>
+                        <option *ngFor="let r of registries" [value]="r.id">{{r.name}}-{{r.url}}</option>
+                    </select>
+                </div>
+            </div>
+            <div class="clr-form-control mt-0" *ngIf="isSystemAdmin && enableProxyCache">
+                <label for="create_project_storage_limit" class="clr-control-label"></label>
+                <div class="clr-control-container input-width">
+                    <div class="space-between" *ngIf=" enableProxyCache && !registries.length" >
+                        <span class="alert-label">{{"REPLICATION.NO_ENDPOINT_INFO" | translate}}</span>
+                        <a class="alert-label go-link" routerLink="/harbor/registries">{{'REPLICATION.ENDPOINTS' | translate}}</a>
+                    </div>
+                </div>
+            </div>
+            <div class="clr-form-control mt-05" *ngIf="isSystemAdmin && enableProxyCache">
+                <label for="create_project_storage_limit" class="clr-control-label"></label>
+                <div class="clr-control-container">
+                    <div class="clr-input-wrapper">
+                        <label class="clr-control-label endpoint">{{ 'PROJECT.ENDPOINT' | translate }}</label>
```

Comment:
![image](https://user-images.githubusercontent.com/29826789/86322786-274cf200-bc6e-11ea-873c-8e175ede67e0.png)
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/29826789/86322877-57949080-bc6e-11ea-913f-92ff54838edd.png)

Reply:
should be consistent  with  the left labels